### DISCUSSION TITLE
dae 中能实现 “域名-IP” 的手工设置吗？

### DISCUSSION THREAD

**Discussion by shookgao** [daeuniverse/dae#286] [Q&A]
比如我想把一个域名：xxx.com 的 IP 解析为一个内网地址：192.168.1.2，dae 能通过配置文件实现吗？

**Comment by wanlce** [comment]
这个不是应该在 hosts 里面改的吗

**Reply by shookgao** [reply]
dae 会读取本地的 hosts 吗？如果会，那就没问题，我晚上可以试试看

**Reply by shookgao** [reply]
试过了，没用

**Reply by mzz2017** [reply]
你这个是 dns 的事情啊，dae 只是代理工具

**Comment by tycn888** [comment]
DNS域名分流 用本地dns服务器解析，
本地DNS服务器设置静态IP，
可以实现功能，
例如域名：xxx.test.local

qname(suffix: test.local) -> alidns

**Reply by thomasdba** [reply]
是可以这么做，但要本地单独启一个dns server，而这个的作用也只限与此，有些浪费了，对于本地个别的一些hosts，能在dae里解决了，就方便多了。
